app/ui-react/pom.xml

```
@@ -283,27 +281,58 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>com.github.eirslett</groupId>
-            <artifactId>frontend-maven-plugin</artifactId>
+            <groupId>com.googlecode.maven-download-plugin</groupId>
+            <artifactId>download-maven-plugin</artifactId>
+            <version>1.3.0</version>
```

```suggestion
            <version>${maven-download.version}</version>
```

Or better yet we could put this in the parent POM.